implementation.md

```diff
@@ -2,4 +2,61 @@
 layout: default
 ---
 
-Work in progress. 
+# Sinopia Technology Implementation
+The Sinopia stack uses [Circle-ci][CI] for continuous integration to
```

Comment:
What's `[Circle-ci][CI]`? Looks like you started to create a link and maybe didn't loop back to add in the URL and correct the syntax?

Reply:
and Circle-CI should be with capital CI

Reply:
Fixed in latest commit.